pkg/config/config.go

```diff
@@ -61,6 +61,7 @@ var (
 //Values for AgentFlavor below
 const (
 	DefaultAgentFlavor = "agent"
+	ClusterAgentFlavor = "cluster_agent"
```

Comment:
This set of constants really start to look like these: https://github.com/DataDog/datadog-agent/blob/master/pkg/aggregator/aggregator.go#L47-L58